src/components/BoardOptionsMenu/BoardOptionsMenu.tsx

```diff
@@ -114,10 +116,12 @@ const BoardOptionsMenu = () => {
           dispatch(updateTitle(result.board.title));
           dispatch(updateAllLists(result.lists));
           dispatch(updateAllCards(result.cards));
-          // TODO add import for tags
+          dispatch(updateAllTags(result.tags));
+          // TODO add import for attachments
         }
-        fr.readAsText(file);
       };
+
+      fr.readAsText(file);
```

Comment:
This was mistakenly inside the onload. This should be outside as it triggers the onload